src/main/java/listeners/InConnection.java

```diff
@@ -0,0 +1,26 @@
+package listeners;
+
+import org.apache.activemq.ActiveMQConnectionFactory;
+
+import javax.jms.Connection;
+import javax.jms.ConnectionFactory;
+import javax.jms.JMSException;
+
+public class InConnection {
+
+
+    private static final ConnectionFactory connectionFactory = new ActiveMQConnectionFactory("tcp://localhost:61616");
+    private static Connection connection;
+
+    static {
+        try {
+            connection = connectionFactory.createConnection();
+        } catch (JMSException e) {
+            e.printStackTrace();
+        }
+    }
```

Comment:
Create connection by first calling, we do not need set up it in static. Any way I do not like static and dynamic blocks))